README.adoc

```diff
@@ -1,48 +1,166 @@
+
+=== MACOS
+- Download the `starfix-macos.zip` file from the Latest Realease
+- Extract and Navigate to `installer` directory
+- On the terminal execute the following(ensure the current directory is `installer` ) 
+
+[source, bash]
+----
+./install
+----
+
+- Once you execute the above command you'll  be prompted for password (for sudo )
+- Test your installation by going to terminal and typing `starfix`
+- To set-up the Browser Extension navigate to this section: https://github.com/starfixdev/starfish/blob/doc/README.adoc#browser-extension[Broswer Extension]
+
+=== BROWSER EXTENSION
+-  Clone This Repository.
+- Now **Load Unpacked Extension** in the `browser-extension` directory.The method for Loading Unpacked extension generally varies  across **browsers** . We'll guide you with loading up on **Google Chrome** and **Mozilla Firefox**.
+
+==== Loading unpacked on Google Chrome
+- Open the Extension Management page by navigating to chrome://extensions.
+- The Extension Management page can also be opened by clicking on the Chrome menu, hovering over More Tools then selecting Extensions.
+- Enable Developer Mode by clicking the toggle switch next to Developer mode.
+- Click the LOAD UNPACKED button and select the extension directory(`browser-extension`).
+Ta-da! The extension has been successfully installed
+
+==== Loading unpacked on Mozilla Firefox
+- Open the `about:debugging` page
+- Click "This Firefox" (in newer versions of Firefox)
+- Click "Load Temporary Add-on"
+- Select any file(like `manifest.json`) in the extension's directory(`browser-extension`).
+
+__**P.S: Browser Extensions will soon be published on Browser's App Store to simplify installation**__ .
+
+== Usage
+ 
+=== As a CLI Tool
+- Launch the terminal and type `starfix https://github.com/user_name/repo_name.git` (You can also use ssh)
+- The repository will be cloned in preferred directory and launched in preferred Editor.See the Demo below:
+image:https://github.com/starfixdev/starfix/blob/master/cli_demo.gif[Starfix CLI,title="Starfix CLI"]
+
+=== Fully Integrated
+- Navigate to the repository (on github.com/user/repo)
+- Click on "Open in IDE" button on Repository page.
+- Repository is cloned in the preferred directory.
+- The cloned Repository is opened in the preferred IDE.
+image:https://github.com/starfixdev/starfix/blob/master/demo.gif[Starfix Demo]
+
+=== Configuration Editor
+- The first time you launch up starfix you'll be prompted to choose preferred Editor and cloning directory.
+- To change the configurations you can launch up terminal and  type `starfix config`
+- You'll then be prompted to choose the preferred Editor and cloning path as shown in Demo below.
+image:https://github.com/starfixdev/starfix/blob/master/config_demo.gif[Starfix Configurations,title="Starfix Configurations"]
+
+== Useful resources:
+
+=== Quick Command reference:
```

Comment:
these are for developing startfix - until now you talked about user info. Some title/intro mixing ?